### PR TITLE
libp2p 6.0.0 release notes blog post

### PR DESCRIPTION
**Pull request by ghost**
After discussing with @whyrusleeping, I'm proposing that we post @Stebalien's [go-libp2p 6.0.0 release notes](https://github.com/libp2p/go-libp2p/blob/master/NEWS.md) to the IPFS blog.  This was a major refactor for go-libp2p and unblocks a huge amount of stuff like QUIC.

The only changes I made from the original source document are:

- Added a few emoji's to try to match the blog's style
- Reduced indent level of everything by one `#` to better distinguish sub-headings from sub-sub-headings
- Listed Steven as the author (since he is)

This is my first post to the IPFS blog, so give me feedback on whether I should make any changes.  More emojis?  Less emojis?  Stylistic changes?

### CODE REVIEW

**Comment by whyrusleeping** [comment]
cc @VictorBjelkholm @Stebalien

File: content/post/39-go-libp2p-6.0.0.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+
+However, we do hope to improve this and get even *better* bandwidth metrics than
+we did before. See [libp2p/go-libp2p-transport#31][] for details.
+
+[libp2p/go-libp2p-transport#31]: https://github.com/libp2p/go-libp2p-transport/issues/31
+
+### Notifications
+
+This release brings performance improvements and easy to reason about ordering
+guarantees libp2p connect/disconnect events:
```
Comment (whyrusleeping) [inline content/post/39-go-libp2p-6.0.0.md]: wording here sounds weird.

File: content/post/39-go-libp2p-6.0.0.md [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+
+1. For any given connection/stream, libp2p will wait for all connect/open event
+   handlers to finish exit before triggering a disconnect/close event for the
+   connection/stream.
+2. When a user calls the Close (or `Reset`) method on a connection or stream,
```
Comment (whyrusleeping) [inline content/post/39-go-libp2p-6.0.0.md]: put Close in backticks?

File: content/post/39-go-libp2p-6.0.0.md (hunk 1 above)
Comment (Stebalien) [reply content/post/39-go-libp2p-6.0.0.md]: weird = missing a word.

How about "This release brings performance improvements and simple ordering guarantees for connect/disconnect events"?

Comment (mgoelzer) [reply content/post/39-go-libp2p-6.0.0.md]: I've taken a stab at rewording this for easier reading.

File: content/post/39-go-libp2p-6.0.0.md (hunk 2 above)
Comment (mgoelzer) [reply content/post/39-go-libp2p-6.0.0.md]: Good catch - fixed.

File: content/post/39-go-libp2p-6.0.0.md (hunk 1 above)
Comment (mgoelzer) [reply content/post/39-go-libp2p-6.0.0.md]: New commit - adopted @Stebalien's wording in latest commit.

**Pull request merged**